Release/src/pch/stdafx.h

```diff
@@ -97,10 +97,12 @@
 #include "cpprest/details/web_utilities.h"
 
 // http
-//#include "cpprest/details/http_helpers.h"
-//#include "cpprest/http_client.h"
-//#include "cpprest/http_headers.h"
-//#include "cpprest/http_msg.h"
+#if defined(RTC_WINDOWS_FAMILY) || defined(RTC_LINUX_FAMILY)
```

Comment:
@chri7325 Should we use our platform defines in 3rd party code?

Reply:
@Craig-Proctor I think it is best. It makes it easier to find where we've needed to have custom code paths. If there is a fix that is upstreamable, at that point we use generic defines and try to upstream it.